lighthouse-viewer/types/viewer.d.ts

```diff
@@ -37,6 +39,14 @@ declare global {
     // Inserted by viewer gulpfile build.
     LH_CURRENT_VERSION: string;
   }
+
+  interface PSIParams {
```

Comment:
Defining it in jsdoc:

```
/**
 * @typedef PSIParams
 * @property {string} url
 * @property {string[]=} category
 * @property {string=} locale
 * @property {string=} strategy
 * @property {string=} utm_source
 */
```

results in:

![image](https://user-images.githubusercontent.com/4071474/63387874-68d8f500-c35b-11e9-8c97-83e47b67241b.png)

Reply:
Refactor that loop (do type checking instead of checking the name of the property)? Or don't do this change? Or some way to define the type in jsdoc and keep the same behavior?

Reply:
oh, ha, that's weird. Somehow the one in d.ts is less strict (setting `value` to `any`) than doing it in a typedef (setting `value` to `string | string[] | undefined`)

Reply:
done.